### ISSUE TITLE
v0.2.2

### ISSUE THREAD

**Issue by LingamYoni** [devapromix/lizardry#99]
Вопрос: зачем нужен "Свиток исцеления" за 140 голды, тратящий 10 маны (всю мою имеющуюся!), если есть "Эликсир исцеления" за 120 голды, делающий ту же работу?))

**Comment by LingamYoni** [comment]
Вдогонку. Или вот еще еда:
![изображение](https://user-images.githubusercontent.com/40486018/202153316-6bab812c-44b5-4d17-a0f7-18dd7a5462ef.png)
Стоит 210 голды, но опять же "Эликсир исцеления" дешевле. Да, еда чуть-чуть восполняет ману, но нафиг она нужна, если есть эликсир маны, который можно заюзать при нехватке маны. Кстати, на скрине бы показать в свойствах еды - что она там делает и сколько восстанавливает маны?

**Comment by LingamYoni** [comment]
А вот баг
![изображение](https://user-images.githubusercontent.com/40486018/202154274-12c6364c-965e-41d9-82d3-3a526f280ed0.png)
После юзанья свитка благословения прямо в таверне через инвентарь, при возврате из него пища приобрела свойства свитка ))

**Comment by LingamYoni** [comment]
И еще вдогонку
![изображение](https://user-images.githubusercontent.com/40486018/202199429-8b9b09d2-5e9c-4c48-8f35-736dc0acc8e9.png)

**Comment by devapromix** [comment]
> После юзанья свитка благословения прямо в таверне через инвентарь, при возврате из него пища приобрела свойства свитка ))

Забавно)) Исправим.

**Comment by LingamYoni** [comment]
> Странно все это)) Как выйдем из этой ситуации?

Как-как... радикально - удалить нафиг этот свиток)) Добавить других, типа свитка благословения, т.е. с полезными свойствами. Кстати, эликсир огра - дает эффект регенерации. А что, сколько, как часто - нет инфы. Раз юзнул его, смотрю в логе каждый (или не?) раунд боя хп поднимается на 1 (sic!) хп - это что за издевательство?))

**Comment by devapromix** [comment]
> > Странно все это)) Как выйдем из этой ситуации?
> 
> Как-как... радикально - удалить нафиг этот свиток)) Добавить других, типа свитка благословения, т.е. с полезными свойствами. Кстати, эликсир огра - дает эффект регенерации. А что, сколько, как часто - нет инфы. Раз юзнул его, смотрю в логе каждый (или не?) раунд боя хп поднимается на 1 (sic!) хп - это что за издевательство?))

Ладно. Свиток под снос)) Насчет регенерации все верно - каждый раунд боя +1hp.

**Comment by LingamYoni** [comment]
> Насчет регенерации все верно - каждый раунд боя +1hp.

Ну, 1хп как-то несерьезно, получается бестолковый эликсир)) Тем более, что эффекты не складываются, лучше уж свиток благословения юзать)

**Comment by devapromix** [comment]
> > Насчет регенерации все верно - каждый раунд боя +1hp.
> 
> Ну, 1хп как-то несерьезно, получается бестолковый эликсир)) Тем более, что эффекты не складываются, лучше уж свиток благословения юзать)

Надо потестировать, сколько нужно добавить за раунд. Не хочется из него сделать суперэликсир))

**Comment by LingamYoni** [comment]
> Надо потестировать, сколько нужно добавить за раунд. Не хочется из него сделать суперэликсир))

Да. Раз уж у тебя все на рандоме завязано)), то можно каждый рандомный раунд прибавлять рандомное (в каких-то рамках от уровня) число хп. А то +1хп каждый раунд - это ниачём)

**Comment by LingamYoni** [comment]
Есть иконки монстров, на которых практически не видно цифр уровня, например:
![изображение](https://user-images.githubusercontent.com/40486018/202634386-ecad3293-1439-4489-af7a-bb7851718754.png)
Надо бы что-то с этим сделать) Контурный шрифт, обводка, подложка, ...?

**Comment by devapromix** [comment]
Сделаем))

**Comment by devapromix** [comment]
Поздравляю)) Но в будущем это не прокатит (я надеюсь))))

**Comment by devapromix** [comment]
И эта, может не допускать к встрече с босом игрока, у которого левел поменьше?

**Comment by LingamYoni** [comment]
Неее, доступ пусть будет, а то дух челленджа исчезнет - хочется ж попрыгать на папку)) Просто сделай их малость (или не малость))) покруче!

**Comment by devapromix** [comment]
Допрыгаешься однако)) Ну ок, жди усиления боссов))

**Comment by LingamYoni** [comment]
Дааа!)

**Comment by devapromix** [comment]
Новая версия с усиленными врагами. Тестим!
